modules/light-clients/08-wasm/types/gas_register.go

```diff
@@ -122,7 +122,7 @@ func NewDefaultWasmGasRegister() WasmGasRegister {
 // NewWasmGasRegister constructor
 func NewWasmGasRegister(c WasmGasRegisterConfig) WasmGasRegister {
 	if c.GasMultiplier == 0 {
-		panic(sdkerrors.Wrap(sdkerrors.ErrLogic, "GasMultiplier can not be 0"))
+		panic(errorsmod.Wrap(errorsmod.ErrLogic, "GasMultiplier can not be 0"))
```

Comment:
You need `ibcerrors "github.com/cosmos/ibc-go/v7/modules/core/errors"`:

```suggestion
		panic(errorsmod.Wrap(ibcerrors.ErrLogic, "GasMultiplier can not be 0"))
```